pipeline/data_merging/aggregate_across_columns.py

```diff
@@ -209,27 +213,32 @@ def selectMaxAlleleFrequency(newRow):
                 maxFreq = aa
                 maxFreqString = "%f (AA from ESP)" % aa
     if newRow["EUR_Allele_frequency_1000_Genomes"] != EMPTY:
-        eur_af = float(newRow["EUR_Allele_frequency_1000_Genomes"])
+        freqs = [float(i) for i in newRow["EUR_Allele_frequency_1000_Genomes"].split(',')]
+        eur_af = max(freqs)
         if eur_af > maxFreq:
             maxFreq = eur_af
             maxFreqString = "%f (EUR from 1000 Genomes)" % eur_af
     if newRow["AFR_Allele_frequency_1000_Genomes"] != EMPTY:
-        afr_af = float(newRow["AFR_Allele_frequency_1000_Genomes"])
+        freqs = [float(i) for i in newRow["AFR_Allele_frequency_1000_Genomes"].split(',')]
+        afr_af = max(freqs)
```

Comment:
Hmm, this brings up an interesting point.  When multiple variants are merged, should the allele frequency be the max or the sum?  This is a good question for a good computational geneticist....